server/database/dbBuild.sql

```diff
@@ -5,16 +5,16 @@ DROP TABLE if EXISTS customer_info CASCADE;
 
 CREATE TABLE customer_info(
   id SERIAL PRIMARY key NOT NULL,
-  first_name VARCHAR(100) NOT NULL,
-  last_name VARCHAR(100) NOT NULL,
+  full_name VARCHAR(100) NOT NULL,
   phone NUMERIC NOT NULL,
   email VARCHAR(50) NOT NULL,
+  zip_code NUMERIC NOT NULL,
   reserve_date TIMESTAMP 
```

Comment:
I would also add an index on reserve_date, because:
1. it will ensure at the database level that we don't allow two bookings at the same time
2. we will want to query by reserve_date, it will speed this up when there are a lot of entries.

Something like `create unique index "index_timestamp" on customer_info using btree(reserve_date);` should work

Reply:
has updated :+1: